packages/components/src/templates/notifications.stories.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../solid-components';
+import { html } from 'lit';
+
+export default {
+  title: 'Templates/Notifications',
+  tags: ['!dev'],
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const InfoNotification = {
+  name: 'Info Notification',
+  render: (_args: Record<string, any>) => {
+    return html`
+      <style>
+        sd-notification.info-variant::part(message) {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+        }
+      </style>
+
+      <sd-notification class="info-variant" variant="info" open>
+        Editing is restricted
+        <sd-button variant="secondary">Request to edit</sd-button>
+      </sd-notification>
+    `;
+  }
+};
+
+export const InfoNotificationInline = {
+  name: 'Info Notification Inline',
```

Comment:
name: 'Info Toast Notification',
  
  Is it possible to show the shadow?
  
  I wanted to have at least 1 toast as a template people can grab ...

Reply:
Added toast as discussed in sync call